src/parlant/modules/examples/parlant_bank.py

```diff
@@ -0,0 +1,46 @@
+from lagom import Container
```

Comment:
Good you have examples but they shouldn't go under `src/parlant/modules/` but rather directly under `examples/modules` in the repo root dir.